simapp/helpers/test_helpers.go

```diff
@@ -78,3 +84,66 @@ func GenTx(gen client.TxConfig, msgs []sdk.Msg, feeAmt sdk.Coins, gas uint64, ch
 
 	return tx.GetTx(), nil
 }
+
+// GenTxWithFeeGranter generates a signed mock transaction with fee granter field set with a given address.
+func GenTxWithFeeGranter(gen client.TxConfig, msgs []sdk.Msg, feeAmt sdk.Coins, gas uint64, chainID string, accNums,
```

Comment:
it seems to me that this function is only used once, in a test. Maybe it would make more sense to just put it as a private function in that test?